Its actually not needed during build. void's build chroot does not come with it.
And installing it by installing `util-linux` breaks the build chroot.
--- a/cmake/XiphosBuildTools.cmake
+++ b/cmake/XiphosBuildTools.cmake
@@ -85,7 +85,6 @@
 
 # ePub help: yelp-build needs uuidgen, epub needs zip
 if (EPUB)
-  xiphos_find_program (UUIDGEN uuidgen)
   xiphos_find_program (ZIP zip)
 endif (EPUB)
 
